app/forms/sufia/forms/batch_edit_form.rb

```diff
@@ -26,9 +26,9 @@ def initialize(model, current_ability, batch)
         def initialize_combined_fields(batch)
           combined_attributes = {}
           permissions = []
-          # For each of the files in the batch, set the attributes to be the concatination of all the attributes
+          # For each of the files in the batch, set the attributes to be the concatenation of all the attributes
           batch.each do |doc_id|
-            work = model_class.load_instance_from_solr(doc_id)
+            work = model_class.find(doc_id).to_solr
```

Comment:
Ah, I slapped this together without a second thought while the builds were broken. Assumed I needed a Solr doc here because of the hash-like `work[key]` invocation on line 34, but perhaps I overthought it. Thx.